aptos-move/transaction-builder-generator/src/rust.rs

```diff
@@ -217,6 +217,9 @@ impl ScriptFunctionCall {
             .with_comments(comments)
             .with_external_definitions(external_definitions)
             .with_serialization(!self.local_types);
+        for (k, v) in script_registry.iter() {
+            eprintln!("reg {}: {:?}", k, v);
+        }
```

Comment:
😭

Reply:
Yeah, that code is not longer called, but better to remove this. Done.